### PR TITLE
Fix Unnecessary Fake Signals

### PR DESCRIPTION
**Pull request by hamed-aloware**
- Fix Unnecessary Fake Signals
- Add back button

### CODE REVIEW

File: resources/js/components/PageNavigationControls.vue [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<template>
+    <div class="card-header d-flex align-items-center justify-content-between" style="border-bottom: 1px solid #e7e7e7;">
+        <div>
+            <button class="btn btn-secondary btn-sm" onclick="history.back()"><</button>
+        </div>
+    </div>
+</template>
```
Comment (amir-the-h) [inline resources/js/components/PageNavigationControls.vue:7]: Add n empty line at the end of the file.

File: resources/js/screens/queues/job-preview.vue [hunk 2]
```diff
@@ -10,9 +11,9 @@
 
             <div class="card-body">
                 <hr>
-                <div><strong>Queue:</strong> {{ this.$route.params.queue }}</div>
-                <div><strong>Partition:</strong> {{ this.$route.params.partition }}</div>
-                <div><strong>Job:</strong> {{ job.name }}</div>
+                    <div><strong>Queue:</strong> {{ this.$route.params.queue }}</div>
+                    <div><strong>Partition:</strong> {{ this.$route.params.partition }}</div>
+                    <div><strong>Job:</strong> {{ job.name }}</div>
```
Comment (amir-the-h) [inline resources/js/screens/queues/job-preview.vue:16]: Revert extra indents

File: src/Repositories/RedisKeys.php [hunk 3]
```diff
@@ -25,6 +25,19 @@ private function queueKey($queue, $prefix = '')
         );
     }
 
+    private function fairSignalKey($queue)
+    {
+        $signal_key_prefix_for_horizon = config('fair-queue.signal_key_prefix_for_horizon');
+        $horizon_prefix = config('horizon.prefix');
+
+        return sprintf(
```
Comment (amir-the-h) [inline src/Repositories/RedisKeys.php:33]: Can you leave an example output as a comment?

**Review by amir-the-h — CHANGES_REQUESTED** [review]
Remove `resources/js/.DS_Store`

File: src/Repositories/RedisKeys.php (hunk 3 above)
Comment (hamed-aloware) [reply src/Repositories/RedisKeys.php:33]: Assume:
```
$queue = 'default'; 
$signal_key_prefix_for_horizon = config('fair-queue.signal_key_prefix_for_horizon'); // "fq:"
$horizon_prefix = config('horizon.prefix'); // "horizon:"
```

Output:
```
horizon:fq:default:[0-9]*
```

It's a key name pattern of fair signals for a specific queue

**Review by amir-the-h — APPROVED** [review]

**Pull request merged**